rkt/pubkey/pubkey.go

```diff
@@ -227,6 +235,36 @@ func downloadKey(u *url.URL) (*os.File, error) {
 	return retTf, nil
 }
 
+func getClient(skipTLSCheck bool) *http.Client {
+	if !skipTLSCheck {
+		return http.DefaultClient
+	}
+	client := *http.DefaultClient
+	var tr *http.Transport
+	// default transport is hidden behind the RoundTripper
+	// interface, so we can't easily make a copy of it
+	if transport, ok := http.DefaultTransport.(*http.Transport); ok {
```

Comment:
Well, what's the circumstance in which `!ok`? I thought the comment on line 250 was implying that this assertion woudl only work on 1.6 or newer.